### ISSUE TITLE
Website - Main Steps Page – Poor Color Choices for those that Have Color Defiance’s (Colorblind)

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#66]
The color of all the Step icons is going to be difficult to see for people that are color deficient (colorblind). To some people, the blues and greens will look the same. The only thing that will help people realize they are different is the icons and numbers in the icons.

See screenshot below.

![complychain - main steps - poor color choice for the icons for those that are colorblind](https://user-images.githubusercontent.com/10423487/30119907-56123f8c-92f5-11e7-993d-4ba238995c4c.png)

**Comment by adriennerlong** [comment]
Thanks for flagging this issue. Since this image has been approved by our leadership, I don't know that we'll be able to change this for the 9/19 release.

**Comment by jfc3-dol** [comment]
@adriennerlong,

It's a risk that you will being taking that people will complain it's not accessible to them depending on their type of colorblindness.

**Comment by jfc3-dol** [comment]
The issues persists over all three platforms (iOS, Android, and web).

**Comment by adriennerlong** [comment]
From ILAB: Our contract with our graphic designer has ended so this is something we are not prepared to address right now. However, we can make this a priority in the next re-vamp of the app (expected 2018) once we have a new contract with our graphic designers.

**Comment by jfc3-dol** [comment]
@adelevie,

Okay, will leave this open so we have it as a reference for the next version of ComplyChain.

**Comment by jfc3-dol** [comment]
The new Main Steps image and messages of what step a person is on now have enough color contrast for the web, iOS, and Android versions of the application.

Closing issue.
